### PR TITLE
chore: add the configuration of action stale for GitHub Action workflow

### PR DESCRIPTION
**Pull request by PenghaiZhang**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change

Migrate use of Stale Bot to actions/stale.

### CODE REVIEW

File: .github/workflows/stale.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-label: 'stale'
+          stale-issue-message: '  This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
```
Comment (edalex-ian) [inline .github/workflows/stale.yaml:13]: Why the two spaces at the start of the message?

**Review by edalex-ian — COMMENTED** [review]
Looks good, just one query though before I approve.

File: .github/workflows/stale.yaml (hunk 1 above)
Comment (PenghaiZhang) [reply .github/workflows/stale.yaml]: I have removed the spaces.

**Review by edalex-ian — APPROVED** [review]
Thanks @PenghaiZhang 

GTG.

**Pull request merged**